Remove payloads flatpak and rpmostree which are not supported by oe.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/anaconda.py | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/pyanaconda/anaconda.py b/pyanaconda/anaconda.py
index b024c0835..6d0d2e819 100644
--- a/pyanaconda/anaconda.py
+++ b/pyanaconda/anaconda.py
@@ -28,7 +28,6 @@ from pyanaconda.core.constants import DisplayModes
 from pyanaconda.core import constants
 from pyanaconda.core.startup.dbus_launcher import AnacondaDBusLauncher
 from pyanaconda.payload.source import SourceFactory, PayloadSourceTypeUnrecognized
-from pyanaconda.payload.flatpak import FlatpakPayload

 from pyanaconda.anaconda_loggers import get_stdout_logger
 stdoutLog = get_stdout_logger()
@@ -82,14 +81,7 @@ class Anaconda(object):
         # Try to find the payload class.  First try the install
         # class.  If it doesn't give us one, fall back to the default.
         if not self._payload:
-            if self.ksdata.ostreesetup.seen:
-                if FlatpakPayload.is_available():
-                    from pyanaconda.payload.rpmostreepayload import RPMOSTreePayloadWithFlatpaks
-                    klass = RPMOSTreePayloadWithFlatpaks
-                else:
-                    from pyanaconda.payload.rpmostreepayload import RPMOSTreePayload
-                    klass = RPMOSTreePayload
-            elif self.opts.liveinst:
+            if self.opts.liveinst:
                 from pyanaconda.payload.livepayload import LiveImagePayload
                 klass = LiveImagePayload
             elif self.ksdata.method.method == "liveimg":
